src/server/string_family.cc

```diff
@@ -543,8 +558,8 @@ SinkReplyBuilder::MGetResponse OpMGet(util::fb2::BlockingCounter wait_bc, uint8_
   }
 
   // Allocate enough for all values
-  response.storage_list = SinkReplyBuilder::AllocMGetStorage(total_size);
-  char* next = response.storage_list->data;
+  response.storage.resize(max(size_t(27), total_size));
```

Comment:
what's 27? :)

Reply:
My birthday date 🙂 Kidding, good that you found it, forgot to add a todo. To prevent sso, because we have a self referential struct, changed it to a `unique_ptr<char[]>`